src/achievements.config.ts

```diff
@@ -14,8 +14,9 @@ export default AchievementSpec({
     ],
   },
   'i18n-reviews': {
-    stat: 'reviews_by_category',
-    category: 'i18n',
+    getCount: ({ reviews_by_category, reviews_by_label }) =>
+      Math.max(reviews_by_category?.docs?.i18n || 0, reviews_by_label?.docs?.i18n || 0) +
+      Math.max(reviews_by_category?.starlight?.i18n || 0, reviews_by_label?.starlight?.i18n || 0),
```

Comment:
Interesting! I was wondering whether the i18n category counted Starlight PRs, too!